app/frontend/src/features/profile/hooks/useLanguages.ts

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from "react-query";
+import { service } from "service";
+
+export const useLanguages = () => {
+  const { data: { languages, languagesLookup } = {}, ...rest } = useQuery(
+    "getLanguages",
```

Comment:
```suggestion
    "languages",
```
We try to stick to nouns for the query keys rather than referring to the function/method name, since it's roughly meant to map some entity in the cache. Also can you extract this to `src/queryKeys.ts` please? We've been trying to do something similar to the routes with these query keys so we can see all the entities we've added/are using React Query for.